frontend/src/components/Authentication/Login.jsx

```diff
@@ -26,18 +29,19 @@ const Login = () => {
   const submitHandler = async () => {
     setLoad(true);
     if (!email || !password) {
-      toast.error("You have to fullfill both email and password");
+      toast.error('You have to fullfill both email and password');
```

Comment:
NIT: fill instead of fullfill. Fullfill seems to be used for fulfill wishes or something like that, plus spelling here might be wrong (you have extra L), though i am not sure